[2024-10-09T05:28:07.229+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Daily_Gold_Report_Generate.daily_report_generate scheduled__2024-10-08T00:00:00+00:00 [queued]>
[2024-10-09T05:28:07.258+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Daily_Gold_Report_Generate.daily_report_generate scheduled__2024-10-08T00:00:00+00:00 [queued]>
[2024-10-09T05:28:07.264+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 4
[2024-10-09T05:28:07.339+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): daily_report_generate> on 2024-10-08 00:00:00+00:00
[2024-10-09T05:28:07.391+0000] {standard_task_runner.py:60} INFO - Started process 750 to run task
[2024-10-09T05:28:07.509+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Daily_Gold_Report_Generate', 'daily_report_generate', 'scheduled__2024-10-08T00:00:00+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/gold_daily_report_generate_pipeline.py', '--cfg-path', '/tmp/tmpxd5d3lx6']
[2024-10-09T05:28:07.512+0000] {standard_task_runner.py:88} INFO - Job 312: Subtask daily_report_generate
[2024-10-09T05:28:07.985+0000] {task_command.py:423} INFO - Running <TaskInstance: Daily_Gold_Report_Generate.daily_report_generate scheduled__2024-10-08T00:00:00+00:00 [running]> on host 9bca14b2d0d7
[2024-10-09T05:28:10.455+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sdeshan' AIRFLOW_CTX_DAG_ID='Daily_Gold_Report_Generate' AIRFLOW_CTX_TASK_ID='daily_report_generate' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-08T00:00:00+00:00'
[2024-10-09T05:28:13.995+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/sklearn/base.py:465 UserWarning: X does not have valid feature names, but StandardScaler was fitted with feature names
[2024-10-09T05:32:04.844+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2024-10-09T05:32:06.671+0000] {process_utils.py:131} INFO - Sending 15 to group 750. PIDs of all processes in the group: [750]
[2024-10-09T05:32:06.685+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 750
